pkg/project/cloud/Taskfile.yml

```diff
@@ -149,6 +150,25 @@ tasks:
     cmds:
       - sh aemw package deploy --file "all/target/*.all-*.zip"
 
+  aem:check:
+    deps: [aem:author:check, aem:publish:check]
+
+  aem:author:check:
+    vars:
+      BASIC_AUTH:
+        sh: sh aemw config value -t "[[.Instance.Config.local_author.User]]:[[.Instance.Config.local_author.Password]]" --output-value value 2>&1
+    desc: check deployed AEM application deployed to instance and dispatcher
+    cmds:
+      - curl -s -u "{{.BASIC_AUTH}}" "http://127.0.0.1:4502/etc/replication/agents.author/publish.test.html" | grep -q "Replication (TEST) of /content successful"
```

Comment:
In theory you can also customize address of the instance in the config file. So either you remove it there (is it really configurable? I can't provide arbitrary ip and port there, right?) or read it from config here too

Reply:
you could, I will make it configurable, bound with each other; now  I am simply  checking what I could do and how...

Reply:
done